head.go

```diff
@@ -698,6 +702,21 @@ func (h *rangeHead) MaxTime() int64 {
 	return h.maxt
 }
 
+func (h *rangeHead) NumSeries() uint64 {
+	return h.head.NumSeries()
+}
+
+func (h *rangeHead) Meta() BlockMeta {
+	return BlockMeta{
+		MinTime: h.MinTime(),
```

Comment:
wonder if anyone is using those `MinTime` / `MaxTime` methods apart here?

Reply:
What do you mean? 

We literally removed this from interface, so if TSDB code base does not use `MinTime`  from outside then it's wrong to leave them as there is no way anyone can use them outside. (:

Reply:
you can still use it when you open a writable database like:
db:=tsdb.Open(...)
blocks:=db.Blocks()
blocks[0].MinTime()


or you mean something different?

Reply:
I can't because you just removed if from interface, righ? ;p

Reply:
We removed the MinTime/MaxTime for the BlockReader interface, but you can still use it on blocks.

Reply:
or you mean remove it for the blocks as well?